src/Resources/views/CRUD/edit_mongo_one_association_script.html.twig

```diff
@@ -52,11 +52,14 @@ This code manage the one-to-many association field popup
 
                 // Maintain state of file inputs
                 $oldForm.find('input[type="file"]').each(function(){
-                    var id = '#'$(this).attr('id');
+                    var id = '#'+$(this).attr('id');
                     $newForm.find(id).replaceWith($(this));
                 });
 
                 $oldForm.replaceWith($newForm);  // replace the html
+
+                Admin.shared_setup(jQuery('#field_container_{{ id }}'));
```

Comment:
can you explain, why are you doing this?

Reply:
Hello! In inline edit mode, ajax request return new form row and this line builds and init costum input type javascript tools. 
Example checkbox style: https://i.stack.imgur.com/UwI5L.png
My English is not good, I'm sorry.

Reply:
Seems lie copy-paste from https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Resources/views/CRUD/Association/edit_one_script.html.twig#L61